### ISSUE TITLE
Validações de models

### ISSUE THREAD

**Issue by vitorp** [vitorp/matricula-web-api#11]
Muitos campos de models estão sem nenhuma validação.

**Comment by Markuus13** [comment]
Posso trabalhar nessa Issue?

**Comment by vitorp** [comment]
Sim, claro. O model slots tem umas validações boas pra começar de data inicial antes da final e dia da semana dentro do range.
